src/lib/app.ts

```diff
@@ -35,7 +35,9 @@ export default async function getApp(
     services: IUnleashServices,
     unleashSession?: RequestHandler,
     db?: Knex,
-): Promise<Application> {
+): Promise<{ app: Application; services: unknown; stores: unknown }> {
```

Comment:
Can we have these as parametric types? Something like `<EServices extends IUnleashServices, EStores extends IUnleashStores>`

```suggestion
): Promise<{ app: Application; services: EServices; stores: EStores }> {
```